BotRoyaleUnity/Assets/Scripts/PartHandler.cs

```diff
@@ -41,6 +45,7 @@ public void delUnattachedParts()
                 rb.mass--;
             }
         }
+        socketIO.EmitCurrentParts(gameObject);
```

Comment:
I recommend moving this out of delUnattachedParts and into the on part destroyed method